packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -133,7 +141,8 @@ contract SystemConfig is OwnableUpgradeable, ISemver {
                 l1StandardBridge: address(0),
                 l2OutputOracle: address(0),
                 optimismPortal: address(0),
-                optimismMintableERC20Factory: address(0)
+                optimismMintableERC20Factory: address(0),
+                gasPayingToken: address(0)
```

Comment:
Will be nice to remove 20+ lines of code when we replace this all with `_disableInitializers`

Reply:
I agree with this altho don't want to couple that into this PR